### PR TITLE
first attempt to have some interaction

### PR DESCRIPTION
**Pull request by mcb**

### CODE REVIEW

File: pivomatic [hunk 1]
```diff
@@ -70,7 +70,19 @@ check-slug() {
...
+  fi
+
+  if [ -z "$RELEASE" ]; then
+    echo "Here's a list of available releases:"
+    pivnet releases -p $SLUG
+    echo "Please choose one and provide the id:"
+    read RELEASEID
+    RELEASE=$(pivnet releases -p $SLUG --format json | jq ".[] | select(.id == ${RELEASEID}).version")
+    echo "Chose $RELEASE"
     exit 1
```
Comment (portepa) [inline pivomatic]: You should remove this line, it kills the process when you provided the release 😉

**Comment by portepa** [comment]
Your code is very good except the `exit 1`. It'll allow users to fetch products/releases without going to pivnet GUI 😄!

I've added functionality to allow user to run `pivomatic` without parameters, as now none are required. You can see the changes here https://github.com/portepa/pivomatic/commit/50aae03a7bdd4ff8a4f3730f3932bb8568d22c06 if you'd like to implement them.

I've also found one issue when providing a wrong slug with `-s a-wrong-slug` which can be fixed with:

```sh
  if [ -z "$RELEASE" ]; then
    ERR_OUTPUT="$(pivnet releases -p $SLUG 2>&1 > /dev/null)"
    [ -z "$ERR_OUTPUT" ] && echo "$SLUG slug does not exists" && exit 1
    echo "Here's a list of available releases:"
    pivnet releases -p $SLUG 
    echo "Please choose one and provide the id:"
    read RELEASEID
    RELEASE=$(pivnet releases -p $SLUG --format json | jq ".[] | select(.id == ${RELEASEID}).version")
    echo "Chose $RELEASE"
  fi
```

Problem with the code above is: it makes 2 useless calls to pivnet -> it should catch `stderr` and `stdout` at the same time. It would also be better if `stderr` would be checked to make sure it contains `Pivnet error: could not find product` and not another error.

**Comment by mcb** [comment]
I now included a check whether that slug exists, without making an extra call to Pivnet. I think I have reduced the calls to a minimum.

**Comment by mcb** [comment]
`exit 1` was there on purpose to help test out without actually kicking off. It's now been removed.

**Comment by mcb** [comment]
@portepa thanks for pointing out the optional params, I included it in the optional check. As for the check if the params have been set, do you want to send in a pull request?

**Comment by mcb** [comment]
Will merge this as the feature has been completed. If we need to iterate once more (due to bugs, performance issues etc. we'll open another issue. Thanks @portepa for your help!

**Pull request merged**